tuning/sft_trainer.py

```diff
@@ -242,9 +249,11 @@ def train(
             "Validation dataset length is %s", len(formatted_validation_dataset)
         )
 
-    callbacks = [FileLoggingCallback(logger)]
-    if is_aim_available():
-        callbacks.append(get_aimstack_callback())
+    file_logger_callback = FileLoggingCallback()
+    if callbacks:
```

Comment:
Should changing 253 like this be better?
   ` if callbacks and file_logger_callback not in callbacks:
`
we only want to append if not passed already?